tools/extensions/generate_extension_db.py

```diff
@@ -15,14 +15,21 @@
 from importlib.util import spec_from_loader, module_from_spec
 from importlib.machinery import SourceFileLoader
 
-BUILDOZER_PATH = os.getenv("BUILDOZER_BIN") or (os.path.expandvars("$GOPATH/bin/buildozer") if
-                                                os.getenv("GOPATH") else shutil.which("buildozer"))
+BUILDOZER_PATH = os.path.abspath(
+    "external/com_github_bazelbuild_buildtools/buildozer/buildozer_/buildozer")
+
+ENVOY_SOURCE = os.getenv('ENVOY_SOURCE', '/source')
+
+if not os.path.exists(ENVOY_SOURCE):
```

Comment:
When doing a local docs build, do we now need to define this explicitly? If so, maybe add to instructins somewhere.

Reply:
only if you call this bazel target directly - the `docs/build.sh` still setss up the paths etc

part of the reason for adding this is because im testing this (and a subsequent PR for moving more docs build to bazel) with a docker image containing (almost) only bazel - which has different paths

i think the main issue im trying to figure out - as noted above - is how best to pass paths into the bazel action as `pwd` is not  `pwd` in the build env i think

ill play with this a bit further to think about the possible paths through this code

Reply:
at very least ill add some comments re significant env vars, and make them more consistent with existing vars

Reply:
comments added at top of script, and paths updated